content/articles/react-flux-frontend.md

```diff
@@ -0,0 +1,65 @@
+Title: A tour of modern frontend with React and Flux
+Date: 2015-08-18
+Short_summary: React and Flux have swept the frontend in the last year or so. Let's look at it.
+Category: Dev
+Authors: Vincent
+
+
+*Reading time: ~15 minutes.*
+
+In this article we are going to have a look at my preferred set of tools for writing React/Flux webapps.
+If you don't know React or Flux, please look at their respective websites: [React](http://facebook.github.io/react/), [Flux](https://facebook.github.io/flux/).
```

Comment:
That sounds slightly condescending IMO. Maybe "or Flux, you can find great docs at their respective websites:" ?